AbstractPyth.sol

```diff
@@ -28,12 +28,35 @@ abstract contract AbstractPyth is IPyth {
         return price;
     }
 
-    function getPrevPriceUnsafe(bytes32 id) external view override returns (PythStructs.Price memory price, uint64 publishTime) {
+    function getLatestAvailablePriceUnsafe(bytes32 id) public view override returns (PythStructs.Price memory price, uint64 publishTime) {
         PythStructs.PriceFeed memory priceFeed = queryPriceFeed(id);
 
+        price.expo = priceFeed.expo;
+        if (priceFeed.status == PythStructs.PriceStatus.TRADING) {
+            price.price = priceFeed.price;
+            price.conf = priceFeed.conf;
+            return (price, priceFeed.publishTime);
+        }
+
         price.price = priceFeed.prevPrice;
         price.conf = priceFeed.prevConf;
-        price.expo = priceFeed.expo;
         return (price, priceFeed.prevPublishTime);
     }
+
+    function getLatestAvailablePriceWithinDuration(bytes32 id, uint64 duration) external view override returns (PythStructs.Price memory price) {
```

Comment:
If you haven't already, can you mention that the docs for these live in `IPyth.sol`? Thanks!

Reply:
I think since these are override people will look at IPyth as the source. (All of the functions in AbstractPyth are undocumented)